src/main/java/io/vertx/core/json/JsonObject.java

```diff
@@ -378,7 +378,7 @@ public static JsonObject of(String k1, Object v1, String k2, Object v2, String k
                               String k4, Object v4, String k5, Object v5, String k6, Object v6,
                               String k7, Object v7, String k8, Object v8, String k9, Object v9,
                               String k10, Object v10) {
-    JsonObject obj = new JsonObject(new LinkedHashMap<>(10));
+    JsonObject obj = new JsonObject(new LinkedHashMap<>(16));
```

Comment:
```suggestion
    JsonObject obj = new JsonObject(new LinkedHashMap<>(14));
```

https://guava.dev/releases/21.0/api/docs/src-html/com/google/common/collect/Maps.html#line.291